src/modules/jcms_rest/src/Plugin/rest/resource/PersonItemRestResource.php

```diff
@@ -71,23 +71,39 @@ public function get($id) {
         if ($research_details_field->get('field_research_expertises')->count()) {
           $research['expertises'] = [];
           $research['focuses'] = [];
+          $expertises = [
+            'id' => [],
+            'name' => [],
+          ];
           foreach ($research_details_field->get('field_research_expertises') as $expertise) {
-            $research['expertises'][] = [
-              'id' => $expertise->get('entity')->getValue()->get('field_subject_id')->getString(),
-              'name' => $expertise->get('entity')->getValue()->toLink()->getText(),
-            ];
+            $expertise_id = $expertise->get('entity')->getValue()->get('field_subject_id')->getString();
```

Comment:
Same as the above comment, this type of method chaining can be dangerous unless you're confident in it.